x-pack/plugins/actions/server/routes/connector/get_all/get_all.ts

```diff
@@ -20,12 +24,18 @@ export const getAllConnectorsRoute = (
   router.get(
     {
       path: `${BASE_ACTION_API_PATH}/connectors`,
-      validate: {},
+      validate: {
+        query: connectorsQuerySchemaV1,
```

Comment:
i could be wrong but does this break backward compatibility? perhaps we have to do something like 

```ts
export const connectorsQuerySchema = schema.maybe(
  schema.object({
    include_system_actions: schema.maybe(schema.boolean()),
  })
);
```

Reply:
This is cherry-picked from `main`. As per issue description:

> Please only refer to [29fa70d](https://github.com/elastic/kibana/pull/175057/commits/29fa70df3708b6b2085f9e59e1eb4478b4b19004) when reviewing. Other code changes were cherry-picked from main

Reply:
Sorry my mistake, was looking at a different file

This shouldn't break backwards compatibility, no. An empty object doesn't break validation